build/templates/_converters.py.mako

```diff
@@ -1,3 +1,9 @@
+<%
+# Have to put this in a variable and add it that way because mako keeps thinking it is for it, not for the output file
```

Comment:
I think you can [do this](https://stackoverflow.com/a/30750520/316875):
```
## -*- coding: utf-8 -*-
```

Reply:
Then mako assumes it is for it and removes it and it ends up not being in the generated file

Reply:
Did you try `##` instead of `#`?

Reply:
Yes.